cumulusci/tests/util.py

```diff
@@ -19,7 +19,7 @@
     UniversalConfig,
 )
 from cumulusci.core.keychain import BaseProjectKeychain
-from cumulusci.tasks.bulkdata.tests import utils as bulkdata_utils
+from cumulusci.tasks.bulkdata.tests.utils import FakeBulkAPI, current_sf_version
```

Comment:
Can we move those here? The current organization feels backwards since these are not intended for use only by the bulkdata tests.

Reply:
It's a mutual import issue that I didn't want to fight with.

Reply:
I can remove the circular import and refactor things in a separate PR.